### DISCUSSION TITLE
Error: Invalid src prop

### DISCUSSION THREAD

**Discussion by Jubayer-cmd** [saleor/saleor#11594] [Q&A]
I am trying to install Saleor storefront on EC2 and when start build command pnpm dev then this error shows in browser

Error: Invalid src prop (http://localhost:8000/media/products/saleor-apple-drink.png) on next/image, hostname "localhost" is not configured under images in your next.config.js See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host

I have created .env.local file and added NEXT_PUBLIC_API_URI but it still tries to load image from localhost:8000

![WhatsApp Image 2022-12-28 at 3 34 00 PM](https://user-images.githubusercontent.com/69860801/209818114-adb60897-d4d6-40d9-acda-4410050a70ed.jpeg)

**Comment by peelar** [comment]
Hello @Jubayer-cmd 👋 

Did you follow [the configuration guide](https://github.com/saleor/react-storefront/blob/canary/apps/storefront/docs/configuration.md) from `storefront`? If so, can you please send over what your `NEXT_PUBLIC_API_URI` looks like?

**Reply by Jubayer-cmd** [reply]
@peelar 
I have added this line "NEXT_PUBLIC_API_URI=http://65.0.182.11:8080/graphql/ "   in .env.local

**Reply by peelar** [reply]
>I have added this line "NEXT_PUBLIC_API_URI=http://65.0.182.11:8080/graphql/ " in .env.local

Are you using [`saleor-platform`](https://github.com/saleor/saleor-platform)? If so, then it's `8000`, not `8080`.

**Comment by alitkp** [comment]
hi, im facing the same problem and ive set NEXT_PUBLIC_API_URI in .env.local 
but i still get:
Error: Invalid src prop (http://localhost:8000/media/products/saleor-apple-drink.png)
any idea what should i do ?
im using saleor-platform btw
